osm2gtfs/creators/managua/managua.json

```diff
@@ -0,0 +1,34 @@
+{
+    "query": {
+        "bbox": {
+            "n": "12.21",
+            "s": "12.04",
+            "e": "-86.12",
+            "w": "-86.38"
+        },
+        "tags": {
+            "route":"bus",
+            "network": "IRTRAMMA",
+            "public_transport:version": "2"
+        }
+    },
+    "agency": {
+        "agency_id": "NI-IRTRAMMA",
+        "agency_name": "Instituto Regulador del Transporte del Municipio de Managua",
+        "agency_url": "http://www.managua.gob.ni/",
+        "agency_timezone": "America/Managua",
+        "agency_lang": "es",
+        "agency_phone": "+505 255-2189",
+        "agency_fare_url": ""
+    },
+    "feed_info": {
+        "publisher_name": "MapaNica.net",
+        "publisher_url": "https://mapanica.net",
+        "version":  "0.1",
+        "start_date": "20171101",
+        "end_date": "20181031"
+    },
+    "timetable": "https://raw.githubusercontent.com/mapanica/pt-data-managua/master/timetable.json",
+    "output_file": "data/ni-managua-gtfs.zip",
+    "selector": "irtramma"
```

Comment:
Why did you choose an folder named "managua"? We were using "irtramma", and I'm almost certain you should then change the selector too (or isn't it necessary at all while using default creator?)

Reply:
Hm, I use Managua since quite some time, because more people can understand what Managua is, rather than IRTRAMMA. Generally I would prefer to have a comprehensive list in the folders for the creators, like proposed also in [this pull request](https://github.com/grote/osm2gtfs/pull/83) here.
* Indeed, as you indicate, it is not necessary to change any selector, because Managua is using the default creators.